src/chttpd/src/chttpd_db.erl

```diff
@@ -932,7 +973,39 @@ send_doc_efficiently(#httpd{mochi_req=MochiReq}=Req, #doc{atts=Atts}=Doc, Header
         send_json(Req, 200, Headers, couch_doc:to_json_obj(Doc, Options))
     end.
 
-send_docs_multipart(Req, Results, Options1) ->
+send_docs_multipart(bulk_get, _Req, Results, Options1, OuterBoundary, Resp) ->
```

Comment:
Start at `Options0` then after the first update make `Options`

Reply:
I think you were working at the same time :-). But yeah I agree, make a separate function as there is not much of a benefit to make it the same function.

Also a single squashed commit rebased on master would be ok.